frontend/src/metabase/admin/settings/components/widgets/ModelCachingScheduleWidget/CronExpressionInput.module.css

```diff
@@ -0,0 +1,54 @@
+.customScheduleLabel {
+  display: block;
+  color: var(--mb-color-text-medium);
+  font-size: 0.875rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  margin-bottom: 6px;
+}
+
+.errorMessage {
+  color: var(--mb-color-error);
+  margin-top: 4px;
+}
+
+.styledInput {
+  width: 100%;
+  border: 1px solid darken(var(--mb-color-border), 10%);
+}
+
+.inputContainer {
+  position: relative;
+}
+
+.infoIcon {
+  position: absolute;
+  right: 1rem;
+  top: 33%;
+  color: var(--mb-color-text-medium);
+}
+
+.infoIcon:hover {
+  color: var(--mb-color-brand);
+}
+
+.popoverContent {
+  background-color: #222;
```

Comment:
In general we should use `--mb-color` color names for consistency. However I'm not sure if we have this dark color in our palette. I guess we'll have to leave it.

Reply:
I was able to remove this by switching to a `Tooltip` component. The background color was really replicating the default tooltip styling